modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,35 +1,31 @@
 = What’s New in Studio
 
-Anypoint Studio 7.13 introduces support for message flow visualization and management of DataWeave libraries through its UI.
+Anypoint Studio 7.14.0 introduces new features to enhance the users' experience making the daily use of Studio easier, faster, and more agile. 
 
-This version also improves the general user experience including new features such as the Exchange Comprehensive Publication.
+== Improved Copy-Paste Experience
 
-== Explore and Debug Shared Flows
+Now, users have the possibility to select if a suffix is added to the copy-pasted Mule Palette elements and to the flows to avoid having similar long names.
 
-Studio now supports message flow visualization of a Mule configuration file loaded from a dependency, which enables exploring and debugging shared flows.
+== Deploy to CloudHub 2.0
 
-image::explore-and-debug-shared-flows.png["A Mule Configuration file inside a dependency is highlighted in the Package Explorer and its flow displayed on the main panel."]
+Aligned with the latest release, Studio allows developers to deploy their applications to https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub 2.0], which offers a wide variety of shared services.
 
-== Easy Management of DataWeave Libraries
+== Project Creation Made Easier
 
-Users now have easier access to manage DataWeave libraries through the Studio UI.
+The on-screen information has been simplified and now users can choose the view of API implementation when creating a project from Studio preferences:
```

Comment:
can choose the view > can customize the view (please apply this suggestion only if it is accurate and reflects the actual the change)

Reply:
No customization. They can only choose to view -or not- the API Implementation section.